.tours/welcome-to-dreamhouse.tour

```diff
@@ -0,0 +1,43 @@
+    },
+    {
+      "title": "Property List - Subscribe",
+      "file": "force-app/main/default/lwc/propertyTileList/propertyTileList.js",
+      "line": 43,
+      "description": "The `propertyTileList` component is where the properties are displayed as a paginated list.\n\nWhen the component loads, it subscribes to the Lightning Message that track property filters. Incoming messages are handled by a `handleFilterChange` function. The function extracts the new filter values from the message and updates the component's local properties."
+    },
+    {
+      "title": "Property List - Wire",
+      "file": "force-app/main/default/lwc/propertyTileList/propertyTileList.js",
+      "line": 34,
+      "description": "Properties are loaded with a `@wire` decorator that calls an Apex endpoint with the filter values.\n\nFilter values are passed with a `$` prefix in the wire configuration to make them reactive. Each time one of these values changes, the wire adapter is reloaded.\n\nThe wire adapter calls a `getPagedPropertyList` Apex method defined in the `PropertyController` class."
+    },
+    {
+      "title": "Property Controller",
+      "file": "force-app/main/default/classes/PropertyController.cls",
+      "line": 45,
+      "description": "The `getPagedPropertyList` method from the `PropertyController` Apex class is where properties are retrieved from the Platform.\n\nThe method builds a SOQL query with the provided filters and returns a paginated list of `property__c` records wrapped in a `PagedResult` object.\n\n`getPagedPropertyList` is exposed to Lightning Web Components thanks to the `@AuraEnabled` annotation. The `cacheable=true` attribute indicates that the returned data is cached.\n\n---\n\nWhile there is a lot to get to know in Dreamhouse, and a lot of code, we'd like to leave off the code tour at this point and suggest you explore the app further in a couple of other ways. \n- One way you can do this is to use our [Quick Start](https://trailhead.salesforce.com/en/content/learn/projects/quick-start-dreamhouse-sample-app) on Trailhead. This will walk you through deploying the app and getting to know it a bit, as well as explore some of the other code samples. \n- But if you prefer to take yourself through the app, you can skip right to the installation instructions that are available in the git repo's [README](https://github.com/trailheadapps/dreamhouse-lwc). From there you can explore the examples bit by bit. There is also an in-app tour that will familiarize you with the app. \n- Lastly, if notice any issue with the app, please open an [issue](https://github.com/trailheadapps/dreamhouse-lwc/issues/new/choose) and let us know.\n\nThanks! And enjoy the Dreamhouse sample app. "
```

Comment:
I'd write "Property__c" with the first letter in uppercase which is the convention that we normally follow in Apex coce

Reply:
Absolutely! The reason why I didn't do that was that the code had it lowercase but since I'm not the only one to notice, I've updated the code and the tour :)